NewForce-Cohort-5/tabloidmvc-the-daily-debuggers
# Bg delete categories

# Description

You can delete categories now from the list.  Simply click the trashcan button next to any category you don't want, and it will disappear.

Fixes # (issue)

## Type of change


# Testing Instructions

1. Clone down the repository.
2. `git fetch -a`
3. `git checkout bg-DeleteCategories`
4. Navigate to the root directory for this repo (directory-to)/tabloidmvc-the-daily-debuggers
5. Start TabloidMVC.sln
6. Make sure your db is created with seed data. Run [01_TabloidMVC_Create_DB.sql](https://github.com/NewForce-Cohort-5/tabloidmvc-the-daily-debuggers/blob/main/SQL/01_TabloidMVC_Create_DB.sql) and [02_TabloidMVC_Seed_Data.sql](https://github.com/NewForce-Cohort-5/tabloidmvc-the-daily-debuggers/blob/main/SQL/02_TabloidMVC_Seed_Data.sql) queries in Visual Studio.
7. Start the server.
8. Log in as admin [admin@example.com](mailto:admin@example.com) (if not already logged in as admin).
9. Click Categories at the top of the page.
10. Click the trashcan to the right of a category you want to delete to remove a category.  I recommend creating a new category and deleting that newly created category to test it.
11. After deleting you should no longer see the category you deleted in the list.



# Checklist:
